src/rise-twitter.js

```diff
@@ -113,19 +115,23 @@ export default class RiseTwitter extends HTMLElement {
         this._play();
       } else {
         console.log('_handlePlay NOT IsAuthorized');
+        if (this.messaging.isConnected()) {
+          this.messaging.sendLicensingWatch();
+        }
         this.eventHandler.emitDone();
       }
     }
   }
 
   _playInPreview() {
+    this.state.setIsPaused(false);
     this.tweet.displayFillerTweets();
-    this.tweet.getTransition().start();
   }
 
   _play() {
+    this.state.setIsPaused(false);
     console.log('_play IsAuthorized');
-    if (this.tweet.getTransition()._isPaused()) {
+    if (this.tweet.getTweets() && this.tweet.getTweets().length > 0) {
```

Comment:
@rodrigopavezi We still need the check for paused state here. It works now but for the future when we want to keep the tweets on the dom it'll break. Perhaps set `this.state.setIsPaused(false);` on line 132 after the check.

Reply:
@jungeunyoon I think we just need to now if there is already tweets to start the transition. The transition will pause when widgets pauses so we don't need to check here as at this time it will be for sure paused. Cheers